com/combat/Fighter.java

```diff
@@ -3,12 +3,14 @@
    String name;
    int str;
    int hp;
+   int money;
```

Comment:
you should add experience for weapons instead of adding this currency system; sad! this is what you should be doing